crawl-ref/source/dat/des/branches/dwarf.des

```diff
@@ -1,3 +1,35 @@
+
+Skip to content
+This repository
+
+    Pull requests
+    Issues
+    Gist
+
+    @yrmvgh
+
+3
+3
+
+    3
+
+jeremygurr/dcssca
+Code
+Issues 42
+Pull requests 1
+Wiki
+Pulse
+Graphs
+dcssca/crawl-ref/source/dat/des/branches/dwarf.des
+141b0a5 18 hours ago
+@yrmvgh yrmvgh Update dwarf.des
+@kilobyte
+@yrmvgh
+@jeremygurr
+@elliptic
+@neilmoore
+@sgrunt
+670 lines (638 sloc) 16.5 KB
```

Comment:
Looks like when I ctrl A ctrl C'd, I accidntally did it to the whole page instead of to the text area.